lib/http/preprocessors.js

```diff
@@ -112,9 +112,12 @@ const authHandler = ({ Sessions, Users, Auth }, context) => {
     if (token == null)
       return;
 
+    const decodedToken = urlDecode(token);
```

Comment:
I just wanted to make sure you saw the standalone comment I left above my review comments. If `context.cookies` automatically URL-decodes, I'm thinking we should just remove the decoding here altogether. Otherwise, we'd be double-decoding.